process/sandbox.md

```diff
@@ -91,3 +91,13 @@ Frequency of reviews can be found on the [repo's README under "Frequency"](https
 ### Annual review
 
 Once in the Sandbox, projects are subject to an [Annual Review](https://github.com/cncf/toc/blob/master/process/sandbox-annual-review.md).
+
+### Sandbox archival
+
+Upon entering sandbox, projects are very early and often experimental. Not all projects that enter the sandbox will leave the sandbox to move on to incubation. Some projects may enter the sandbox and not grow or see widespread adoption or interest. This could be the result of a number of factors, such as being too early in the market, the problem targeted is being addressed in other ways, etc.
+The following criteria are leveraged by the TOC to identify when a Sandbox project is no longer viable and is not expected to move to incubation. The TOC will consider any of the criteria or a combination of them for Archival evaluation.
+* Missed Annual review by more than two months
+* Stale contributions over the course of 18 months - stale contributions include non-significant contributions (minor changes like color in the UI, or grammar corrections), lack of contributions, little discussion or activity on issues, etc. 
+* Missed reasonable [milestones] - subject to the nature of the project, project may miss milestones a reasonable person would assume to have been completed considering other factors about the project. 
```

Comment:
I don't think the anchor will resolve.

Reply:
Does this wording not turn milestones into requirements?  Who is the "reasonable person" in this situation? (Consider, if you will, a project that wishes to remain in the CNCF, or at a particular level, against the will of a section of the community to remove it.)

Is it the intention to also define the vote required to archive a project?  Will it be two-thirds, as with a project joining?

Reply:
Thats a good call out and not what is intended. Milestones are guiding points, but not requirements. If the annual review is missed, contributions have become stale, they're also not likely hitting some of those milestones either, I'm going to adjust this to make that more apparent. Thanks!